test/cpp/interop/client.cc

```diff
@@ -81,6 +81,14 @@ DEFINE_string(default_service_account, "",
 DEFINE_string(service_account_key_file, "",
               "Path to service account json key file.");
 DEFINE_string(oauth_scope, "", "Scope for OAuth tokens.");
+DEFINE_bool(do_not_abort_on_transient_failures, false,
```

Comment:
nit: the flag name is long and has a not in it. Maybe remove do_not_ and change the default value?